services/minespace-web/src/components/dashboard/mine/incidents/Incidents.js

```diff
@@ -134,7 +134,7 @@ const Incidents = (props) => {
           >
             Click here
           </a>
-          for more information on mining incidents and dangerous occurrences in British Columbia.
+          for more information on reportable incidents.
```

Comment:
It does bother me that IncidentForm has "Please add the final report documentation by clicking below" (no period at end) and this one has "Click here for more information on reportable incidents." (period at end of statement). Do we know for sure which style is preferred?

Reply:
Hmmm, yes, that is inconsistent.  I'm not sure if there is a preferred style here.  I think removing the periods would be the way to go
<img width="502" alt="image" src="https://user-images.githubusercontent.com/83598933/217616965-f3a5536b-7440-4168-a1e7-3b6b322e4403.png">